### ISSUE TITLE
Cheapest hour time range

### ISSUE THREAD

**Issue by swepan** [dala318/nordpool_planner#22]
Is it possible to have a sensor that shows calculated cheapest hours?
Ex. cheapest hours start at 21:00 and last until 03:00.
Like attribute starts_at.

**Comment by dala318** [comment]
For sure, can have a look at it.
But can also be solved by a template sensor 

```
- sensor:
    - name: "Sheap at"
      unique_id: electricity_sheap_at
      state: "{{ state_attr('binary_sensor.nordpool_planner_entity_id', 'start_at') }}"
```

**Comment by dala318** [comment]
For sure, can have a look at it.
But can also be solved by a template sensor 

```
- sensor:
    - name: "Sheap at"
      unique_id: electricity_sheap_at
      state: "{{ state_attr('binary_sensor.nordpool_planner_entity_id', 'start_at') }}"
```

Since create individual binary_sensors by the settings it's not obvious how to copy those settings to value sensors.

**Comment by dala318** [comment]
As each nordpool_panner entity is configured manually as a binary_sensor there is no intuitive way to create these sensor cloned with numerical date-time values. Could be solved together with #23. Until then the template-sensor above will be the easiest solution.
